packages/react-native/ReactAndroid/src/main/java/com/facebook/react/modules/toast/ToastModule.kt

```diff
@@ -32,15 +32,15 @@ internal class ToastModule(reactContext: ReactApplicationContext) :
   override fun show(message: String?, durationDouble: Double) {
     val duration = durationDouble.toInt()
     UiThreadUtil.runOnUiThread {
-      Toast.makeText(getReactApplicationContext(), message, duration).show()
+      Toast.makeText(reactApplicationContext, message, duration).show()
```

Comment:
Are these changes related to the migration? If not, let's please take them out – preferably you could send them over in a separate PR

Reply:
I apply your code review! Thanks for review it.